routes/api.php

```diff
@@ -36,8 +36,9 @@
 Route::delete('/lockers/{lockerGuid}/claims/{claimId}', 'LockerClaimController@destroy')->name('lockers.claims.destroy');
 
 Route::post('/lockers/{lockerGuid}/claims/{claimId}/setup', 'LockerClaimController@setup')->name('lockers.claims.setup');
-Route::post('/lockers/{lockerGuid}/claims/{claimId}/reset/key', 'LockerClaimController@resetKey')->name('lockers.claims.reset.key');
+Route::post('/lockers/{lockerGuid}/claims/{claimId}/update-key', 'LockerClaimController@updateKey')->name('lockers.claims.change.key');
```

Comment:
I can't recall if there's a place where we use route aliases, so we might as well remove it, but in case we don't: replace `change` with `update`.